### DISCUSSION TITLE
Education Github Benefits Access

### DISCUSSION THREAD

**Discussion by adityahere7** [community/community#135350] [GitHub Education]
### Hi everyone,

When can i access my GitHUB Education Benefits. Its already more than 3 days since i got approved

**Comment by davevad93** [comment]
Hi @adityahere7 ,

The waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @adityahere7 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
